multimedia/xupnpd/Makefile

```diff
@@ -1,27 +1,19 @@
-#
-# Copyright (C) 2013-2017 OpenWrt.org
-#
-# This is free software, licensed under the GNU General Public License v2.
-# See /LICENSE for more information.
-#
-
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=xupnpd
-PKG_REV:=e4e542d9b6d0043d470fda283e2cd325bbb91950
-PKG_VERSION:=2018-11-20
-PKG_RELEASE:=2
+PKG_REV:=2bc1e741e0efe04cb3150430ff25410093618b4f
+PKG_VERSION:=2021.04.08
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/clark15b/xupnpd/tar.gz/$(PKG_REV)?
```

Comment:
Switching to local tarballs should actually fix CI.

Reply:
The CI is failing because https://buildbot.aparcar.org/apk.static is down.

Reply:
Local tarballs done in 726b379b17162db2b6e1f73cb83148e0b5099a11